dev-lang/fpc/fpc-3.0.4.recipe

```diff
@@ -0,0 +1,145 @@
+SUMMARY="Professional Pascal compiler"
+DESCRIPTION="Free Pascal is a 32, 64 and 16 bit professional Pascal compiler. \
+It can target many processor architectures: Intel x86 (including 8086), AMD64\
+/x86-64, PowerPC, PowerPC64, SPARC, ARM, AArch64, MIPS and the JVM. \
+Supported operating systems include Linux, FreeBSD, Haiku, macOS/iOS/\
+iPhoneSimulator/Darwin, DOS (16 and 32 bit), Win32, Win64, WinCE, OS/2, \
+MorphOS, Nintendo GBA, Nintendo DS, Nintendo Wii, Android, AIX and AROS. \
+Additionally, support for the Motorola 68k architecture is available in the \
+development versions."
+HOMEPAGE="https://freepascal.org/"
+COPYRIGHT="1993-2015 Florian Klaempfl and others"
+LICENSE="
+	GNU LGPL v3
```

Comment:
Hmm, I saw this earlier but forgot to leave a comment about  it.
There are two possible fixes here:
* without alphabetical sorting:
```diff
-LICENSE="
-	GNU LGPL v3
+LICENSE="GNU LGPL v3
	GNU GPL v3"
```
* with alphabetical sorting:
```diff
-LICENSE="
-	GNU LGPL v3
+LICENSE="GNU GPL v3
-	GNU GPL v3"
+	GNU LGPL v3"
```
If you think most of the sources are covered by the LGPL then it's probably OK to keep `LICENSE` unsorted.
Otherwise I'd say that sorting would be better.
(If you copy/paste, don't forget to get rid of the white spaces, as we should only have tabs here.)

Reply:
Only the compiler is GNU GPL. Every thing else (especially what you can link in an application) is LGPL (Runtime library, packages). So, this order make sense, even if it was not done on purpose ;-)
More details here : https://freepascal.org/faq.var#general-license

Reply:
Okay, so let's keep it with LGPL first.
Can you still apply the style change to obtain this?
```
LICENSE="GNU LGPL v3
	GNU GPL v3"
```
Thanks!